python/cutlass/backend/utils/device.py

```diff
@@ -78,7 +78,7 @@ def device_cc(device: int = -1) -> int:
         device = cutlass.device_id()
 
     if cutlass._use_sycl:
-        # Using '11' to encode Intel PVC as an integer in the expected format.
+        # Using '11' to encode Intel Xe as an integer in the expected format.
```

Comment:
Is this case (and other commenty in python files) still specific to PVC or is it also general across all Xe HW?